config/settings/tailer.q

```diff
@@ -50,7 +50,9 @@ eodwaittime:0D00:00:10.000
 tpcheckcycles:0W                                                                            // number of attempts to connect to tp before process is killed
 
 \d .ds
+datastripe:0b
 period:0D01
+lasttimestamp:.z.p-4*period
 
```

Comment:
are these changes relevant to this PR? I don't understand why they're here

Reply:
These lines have now been removed